### DISCUSSION TITLE
Stop loading model-viewer in mobile browser

### DISCUSSION THREAD

**Discussion by mpulock** [google/model-viewer#1905] [Q&A]
Hi,
 
is there any way to stop Model-viewer from loading in mobile browsers? I would like to load it in desktop browsers but, not in mobile till I can find a way to improve the page performance.

Thank you.

**Comment by elalish** [comment]
You can use any of the normal web techniques to only show model-viewer on certain types of browsers. You may want to lazy-load model-viewer.js, so that it only downloads if you're on the desired browsers. There are many ways to achieve this, and none are specific to model-viewer.
